.github/workflows/pulsar-ci.yaml

```diff
@@ -198,6 +198,10 @@ jobs:
           - name: Pulsar IO
             group: PULSAR_IO
             timeout: 75
+          - name: Pulsar IO - Elastic Search
```

Comment:
they take 14min+12min while the main Pulsar IO still takes 45 minutes. 
Could you put Elastic and Kafka together to save one runner?

Reply:
Another reason is that I have concerns that it's the Elastic Search jobs that sometimes stall. When we have smaller builds, it will be easier to investigate.